packages/hoc-input/src/formInput.js

```diff
@@ -73,13 +81,38 @@ const formInput = WrappedComponent => {
         onFocus: this.onFocus,
         onBlur: this.onBlur
       };
+
+      const isSearch =
+        inputProps.type && inputProps.type.toLowerCase() === "search";
+      const composedProps = isSearch
+        ? { ...searchProps, ...inputProps }
+        : inputProps;
+      const icon = composedProps.icon && (
+        <div className="icon">
+          <i className={composedProps.icon} />
+        </div>
+      );
+      const clearButton = isSearch &&
+        this.state.value && (
+          <Link className="clear" onClick={this.handleClearClick}>
+            <i className="wg-close-int" />
```

Comment:
Close button icon should be `GREY_32` 
![image](https://user-images.githubusercontent.com/17936244/35273903-5ac30fb8-0021-11e8-8e6f-e65cff60ab28.png)

Set `cursor: pointer` to `i` tag.

Reply:
It was `cursor:pointer` because it's inside a link. I've changed to `cursor:default` as that's how it works on the native search input. Should I use `pointer` even so?

Reply:
Yeah. If you position the mouse on the left of the icon, it is a pointer. If you go over the icon, it s default.
We should keep consistency here.